pyxrf/model/load_data_from_db.py

```diff
@@ -305,8 +442,12 @@ def get_total_scan_point(hdr):
 
 
 def map_data2D_srx(runid, fpath,
```

Comment:
I am not sure if it is possible right now. It may probably be implemented when Databroker 1.0 is deployed, which can assemble data from multiple Xspress3 HDF5 files into ``xarray``. Currently the only way to assemble data is to go through the events, read data for individual line scans and assemble it into the table, and it is done differently for different beamlines. It also may happen unification of reading procedure will require revision of some plans at the beamlines so that the structure recorded data falls into some defined pattern and only naming convention is different.